AuxTel/AuxTel-Standard-Operations/AuxTel-Nighttime-Operations/Shutdown.rst

```diff
@@ -31,7 +31,7 @@ The Scheduler automatically adds new targets to the queue until the start of nau
 If there are no targets, the Scheduler will stop adding new ones.
```

Comment:
I was unable to leave comments on certain lines above on this procedure so I will leave them here:
Lines 16 and 19 are duplicated: "...ATCS and LATISS CSCs should be set on \`\`STANDBY\`\`
Line 101: the link to the mentioned document seems to be empty.
Line 132: After parking ~~the~~ AuxTel and taking daytime calibration, the LATISS CSCs are still in  \`\`ENABLED\`\` state.
Line 144: \*#rubin_daytime_tasks\* and \*#summit_auxtel\*